index.html

```diff
@@ -0,0 +1,88 @@
+      <li>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor   ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</li>
+      <li>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</li>
+    </ul>
+    </footer>
+    <script>
+      var counter = 0;
+      var userInput = prompt ("What is your name?");
+      alert ("Welcome " + userInput +"!");
+      while (counter < 7) {
+      var response1 = prompt ("Do I have any pets?").toLowerCase();
+      if (response1 === "no") {
+        alert("Correct! I do like pets though!");
+        console.log("Correct! I do like pets though!");
+        counter++;
+      } else {
+        alert("Sorry! I don't have any pets...yet! ");
+        console.log ("Sorry! I don't have any pets...yet! ");
+      }
+      var response2 = prompt ("Is math my favorite subject?").toLowerCase();
+      if (response2 === "no") {
+        alert("Sorry! Math is my favorite subject!");
+        console.log("Sorry! Math is my favorite subject!");
+      } else {
+        alert("Correct! Math is awesome!");
+        console.log("Correct! Math is awesome!");
+        counter++;
+      }
+      var response3 = prompt ("Was I born in Croatia?").toLowerCase();
+      if (response3 === "no") {
+        alert("Correct! I was born in Serbia.");
+        console.log("Correct! I was born in Serbia.");
+        counter++;
+      } else {
+        alert("Actually, I was born in Serbia!");
+        console.log("Actually, I was born in Serbia!");
+      }
+        alert("You've got " + counter + " out of 3 correct!");
+        break;
+    }
+
+// Guess_my_number game
+
+      var counter = 1;
+      var userNum = parseInt(prompt("Guess my number"));
+
+      while (counter < 3) {
+      userNum = parseInt(prompt("Guess my number"));
+        console.log("The counter is at " + counter);
+        if(userNum === 24) {
+        alert("You got it!");
+        break;
+      } else if (userNum > 24) {
+        alert("Your number was too high");
+        counter++;
+        console.log("The counter is at " + counter);
+      } else {
+        alert("Your number was too low");
+        counter++;
+        console.log("The counter is at " + counter);
+      }
+      }
+
```

Comment:
Your javascript should be in a separate script file named game.js and linked before the closed body tag.